source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required (VERSION 3.5)
+set(CMAKE_OSX_SYSROOT iphonesimulator11.2)
+add_compile_options(-std=c++14 -Wall)
+project (AdaptiveCardsLib)
+include_directories(../../../../shared/cpp/ObjectModel)
+#find_library(UIKIT 
+#    UIKit
+#    HINTS /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/System/Library/Frameworks/
+#    )
+#find_library(FOUNDATION Foundation)
+#message(STATUS "UIKIT_Library_DIR: ${UIKIT}")
+#message(STATUS "Foundation_Library_DIR: ${FOUNDATION}")
+#FIND_PATH(UIKIT_INC UIKit/UIKit.h
+#    HINTS /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/System/Library/Frameworks/
+#    )
+#include_directories(${UIKIT_INC})
+#message(STATUS "UIKIT_INC: ${UIKIT_INC}")
+#message(STATUS "sysroot: ${CMAKE_OSX_DEPLOYMENT_TARGET}")
+#MARK_AS_ADVANCED(UIKIT_LIBRARY)
+#SET(EXTRA_LIBS ${UIKIT_LIBRARY})
```

Comment:
Doesn't look like you need these commented-out sections.

Reply:
sorry, I will clean them up

---
In reply to: [165780511](https://github.com/Microsoft/AdaptiveCards/pull/1097#discussion_r165780511) [](ancestors = 165780511)